src/components/grid-aware/HomePageLargeParagraph/HomePageLargeParagraph.module.css

```diff
@@ -0,0 +1,59 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  justify-items: center;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+  padding-bottom: 120px;
+  padding-top: 120px;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+  text-align: center;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
```

Comment:
In the designs, it looks like it actually has 20px of left and right spacing. Could you update this?

Reply:
👍 You're right. Changed from 40px to 20px.